pkg/tarutil/tarutil.go

```diff
@@ -277,7 +237,7 @@ func (r *TarReadCloser) Close() error {
 // Gzip/Bzip2/XZ detection is done by using the magic numbers:
 // Gzip: the first two bytes should be 0x1f and 0x8b. Defined in the RFC1952.
 // Bzip2: the first three bytes should be 0x42, 0x5a and 0x68. No RFC.
-// XZ: the first three bytes should be 0xfd, 0x37, 0x7a, 0x58, 0x5a, 0x00. No RFC.
+// XZ: the first six bytes should be 0xfd, 0x37, 0x7a, 0x58, 0x5a, 0x00. No RFC.
```

Comment:
Why this has changed?

Reply:
I think "three" was a typo. It is six bytes